docker/prod/docker-entrypoint.sh.socialhome

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Socialhome Docker Production entrypoint
```

Comment:
This should be able to run circus if we go by that - no need to have uwsgi/daphne/runworker/rqworker/rqscheduler even available as options imho.